### PR TITLE
remove deprecated parameters

### PR DESCRIPTION
**Pull request by goodfeli**

### CODE REVIEW

**Comment by npapernot** [comment]
LGTM

File: cleverhans/utils_tf.py [hunk 1]
```diff
@@ -162,8 +153,7 @@ def model_train(sess, x, y, predictions, X_train, Y_train, save=False,
                 train_step.run(feed_dict=feed_dict)
             assert end >= len(X_train)  # Check that all examples were used
             cur = time.time()
-            if verbose:
-                _logger.info("Epoch " + str(epoch) + " took " +
+            _logger.info("Epoch " + str(epoch) + " took " +
                              str(cur - prev) + " seconds")
```
Comment (catherio) [inline cleverhans/utils_tf.py]: This line is now overindented, so PEP8 is complaining.

**Comment by catherio** [comment]
Linting bug was masking a real failure. A test needs to be updated to not exercise the deprecated API:

```
======================================================================
ERROR: test_mnist_blackbox (test_mnist_blackbox.TestMNISTBlackboxF)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/travis/build/tensorflow/cleverhans/tests_tf/test_mnist_blackbox.py", line 27, in test_mnist_blackbox
    report = mnist_blackbox.mnist_blackbox(**mnist_blackbox_args)
  File "/home/travis/build/tensorflow/cleverhans/cleverhans_tutorials/mnist_blackbox.py", line 225, in mnist_blackbox
    rng=rng)
  File "/home/travis/build/tensorflow/cleverhans/cleverhans_tutorials/mnist_blackbox.py", line 71, in prep_bbox
    args=train_params, rng=rng)
TypeError: model_train() got an unexpected keyword argument 'verbose'
```

**Comment by goodfeli** [comment]
my latest change is big enough it should probably be reviewed again

**Comment by npapernot** [comment]
I took a look at the changes again and they still look good, thanks for taking care of this

**Comment by goodfeli** [comment]
thanks

**Pull request merged**